packages/core/src/state-management/state-manager.ts

```diff
@@ -202,7 +202,12 @@ export class StateManager {
    * @param tip - Stream Tip CID
    * @private
    */
-  update(streamId: StreamID, tip: CID): void {
+  update(streamId: StreamID, tip: CID, model?: string): void {
```

Comment:
There's no need for this function to take a `model` argument.  This function is for applying the new update to the StreamState for the updated stream - the model property doesn't affect that behavior at all.  For now, nodes that receive UPDATE messages can just ignore the model field entirely.  It'll be used later by the indexer but that isn't hooked up yet.